lib/pure/complex.nim

```diff
@@ -7,8 +7,31 @@
 #    distribution, for details about the copyright.
 #
 
-## This module implements complex numbers.
-## Complex numbers are currently implemented as generic on a 64-bit or 32-bit float.
+## This module implements complex numbers
+## and basic mathematical operations on them.
+##
+## Complex numbers are currently generic over 64-bit or 32-bit floats.
+
+runnableExamples:
+  from math import almostEqual, sqrt
```

Comment:
```suggestion
  from std/math import almostEqual, sqrt
```

Reply:
Should I also change the imports in the module itself and the tests or would that cause problems?

Reply:
I don't think that would cause problems(you could give a try). `import std/math` is prefered to `import math`. I don't care about the old parts. But I think the newer addition should use `std` namespace.